packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -237,11 +260,16 @@ export default function OrgNav({
                   </OrgNavLink>
 
                   <Menu
-                    open={open}
-                    setOpen={setOpen}
+                    open={accessManagerOpen}
```

Comment:
I'm not sure I understand this change. Isn't this menu for both AM and Atlas?

Reply:
I think that this is just cleaning up variable names,  to avoid confusion about which indicate the on prem user dropdown is open vs. the access manager is open, etc.

Reply:
there are two menus on the page that we're controlling instead of just one

Reply:
so had to be more specific w naming